docs/source/algorithms/CalculateCarpenterSampleCorrection-v1.rst

```diff
@@ -0,0 +1,174 @@
+﻿.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
```

Comment:
Needs an extra comment about this being in-plane only. A future PR will add the out-of-plane code.